java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSigningIT.java

```diff
@@ -65,6 +74,51 @@ public void afterClass() throws IOException {
         IntegrationTestConfigContext.cleanupTestDirectory(mantaClient, testPathPrefix);
     }
 
+
+    // When MantaClient TLS security is disabled (such as when testing against a
+    // lab instance without a self signed cert), we will also need to refrain
+    // from checking the certs on the resulting signed urls.
+    private HttpURLConnection openHttpURLConnection(URI uri, MantaClient client) throws IOException {
+        boolean tlsInsecure = client.getContext().tlsInsecure();
+        HttpURLConnection connection = (HttpsURLConnection)uri.toURL().openConnection();
+        if (tlsInsecure && connection instanceof HttpsURLConnection) {
+            HttpsURLConnection sConn  = (HttpsURLConnection)connection;
+
+            TrustManager[] trustAllCerts = new TrustManager[] {new X509TrustManager() {
+                    @Override
+                    public X509Certificate[] getAcceptedIssuers() {
+                        return null;
+                    }
+                    @Override
+                    public void checkClientTrusted(X509Certificate[] certs, String authType) {
+                    }
+                    @Override
+                    public void checkServerTrusted(X509Certificate[] certs, String authType) {
+                    }
+                }
+            };
+
+             HostnameVerifier hostnameVerifier = new HostnameVerifier() {
```

Comment:
This code block could be replaced with a lambda

```
HostnameVerifier hostnameVerifier = new HostnameVerifier() {
                @Override
                public boolean verify(String hostname, SSLSession session) {
                    return true;
                }
            };
```
to 

`HostnameVerifier hostnameVerifier = (hostname, session) -> true;`